src/metabase/db/data_migrations.clj

```diff
@@ -384,6 +385,13 @@
                                  [:like
                                   :dashcard.visualization_settings "%\"click_link_template\":%"]]})))
 
+(defmigration ^{:author "tsmacdonald", :added "0.41.0"} add-admin-group-root-moderate-entry
+  (binding [perms/*allow-admin-permissions-changes* true]
+    (u/ignore-exceptions
+      (db/insert! Permissions
+        :group_id (:id (perm-group/admin))
+        :object   "/collection/root/moderate/"))))
+
```

Comment:
these `defmigration`s are always not great if there's any other way. Could you use a normal migration and update the admin group with this Permission and ensure that when created initially with `(perm-group/admin)` (which can create it if it doesn't exist) it adds the Permission here? That way you either add it if the perm group exists, or don't add it in the migration but creating the admin permission group ensures this permission exists. As far as i can tell the id should be where the PermissionGroup name is `"Administrators"`

Reply:
The permissions system is prefix-based in the sense that if you have permissions to `/some/thing/` then you also have perms to anything starting with `/some/thing/`, e.g. `/some/thing/else/`. Admins all have permissions to `/`, which implies `/collection/root/moderate`... this line shouldn't be needed.

Seconding what Dan said about adding `defmigrations` as well -- we really really really shouldn't add them unless we absolutely can't do it any other way -- you could pretty easily do this as single SQL query if you needed to and put it in Liquibase

Reply:
:sparkles: removed, `/` permission will be adequate